frontend/components/homepage/layout/Navigation.tsx

```diff
@@ -6,7 +6,7 @@ import { Acme } from "next/font/google";
 import clsx from "clsx";
 import { RoutRoute, SocialRoute } from "../route";
 import { NavigationMenu } from "./NavigationMenu";
-import { NavigationDrawer } from "./NavigationDrawer";
+import { NavigationDrawer } from "./navigationDrawer";
```

Comment:
in the dev this bug has been fixed, it is better to not change anything irrelevant to your ticket in your PR. You can run "git merge origin dev" to merge the latest code from dev into your branch to solve this problem.